Agile_process.md

```diff
@@ -0,0 +1,287 @@
+be a UI campaign.
+
+## Process for proposing new sprints or campaigns
+
+To propose a new sprint, the process to follow is:
+
+1. Open a new issue on JIRA, and assign to it the issue type "Proposed Sprint" (we don't have a separate type for sprints and campaigns):
+
+![Proposed sprint screenshot](images/jira-proposed-sprint.png)
+
+2. Provide in the *Summary* field a one sentece description of the sprint.
+
+2. Fill out the *Description* field with a body of text according to the template in the section immediately below.  It should be relatively concise, but try to answer each of these questions clearly, so that the group leads and KBase executive committee can use this information to decide whether to accept your proposal and put it into the sprint pipeline.
+
+2. Assign it to Shane Canon, the production lead.
+
+2. List any components you may think it impacts, if you know in advance. 
+
+2. At this time, don't bother adding Epic/Theme or Sprint labels yet, as those will be added if the sprint is actually accepted for execution later on.
+
+### Template
+
+**Why it’s important:** Explain why KBase should do this.  How will it
+help make KBase unique, bring in users, improve the system, etc.  Why is
+it more important for KBase to do this compared to other priorities.
+
+**What needs to be done?** Provide a brief paragraph describing what has
+to be done including new data types, new UI pieces, example, etc.  Be
+complete but avoid going into implementation details.  This is more
+about understanding the scope of the activity versus exactly how it will
+be done.
+
+**Estimated Number of Sprints:** X sprints
+
+**Key Staff:** List important staff needed for the Sprint and what
+specific skills and knowledge they provide.
+
+**Dependencies:**  List any other campaigns, sprints, or activities
+that impact your proposed sprint or campaign.
+
+### Evaluation of sprint proposals
+
+Once a sprint/campaign proposal is in, there may be discussion and questions about it directly on the JIRA ticket.  The members of the KBase Executive Committee, who ultimately will make the decisions on which sprints to put into the implementation pipeline, will be added as watchers to the ticket so they have a chance to comment.
+
+Then, during the next maintenance week, the EC will include in its evaluation and re-prioritization decisions (that normally consider the outcome of the just-finished sprint cycle) any new proposals that have arrived.  This will allow the EC to weigh these proposals in the context of the outcomes and feedback of the recently concluded sprint cycle.
+
+The EC can decide to assign the proposals either to immediate execution in the next cycle, to be put in the queue for a later cycle, or it can decline the proposal.
+
+If a proposal is accepted, it will be given a sprint label and an epic/theme in JIRA so it can be sorted accordingly, and it will be re-assigned to its ultimate product owner.
```

Comment:
Still, we can refine it here, the point of a good PR discussion is actually to have a chance to iterate over our processes as we use them, and improve them if we see a chance to do so...

Given that for the Issue type, JIRA's agile plugin automatically creates "Epic" as a type, we might as well go with that, as @nlharris says:

![image](https://cloud.githubusercontent.com/assets/57394/10114933/425def58-63af-11e5-826b-44c267e534d7.png)

But after that, we do have in addition, two separate label fields, 'Epic/Theme' label, *and* 'Sprint' label.

So my question is really for @scanon: after running a few cycles, do you think we really need both types of labels?  Initially we figured we'd want to be able to label each campaign, and also be able to label by sprint as well.

But honestly, I'm starting to worry that it's a proliferation of too many management knobs, and too much information to track.

The way these labels can be used is only by *exact* match. On the other hand, they represent a separate search field in a JQL (the "JIRA SQL" pseudo-language) query, such as this one that searches for the [sprint label 'euk-backlog'](https://atlassian.kbase.us/browse/NAR-812?jql=Sprint%20%3D%20euk-backlog).

What do people think about this?